submissions/AntonKottans/html-css-popup/style.css

```diff
@@ -0,0 +1,172 @@
+* {
+	margin: 0px;
+	padding: 0px;
+}
+
+body {
+	min-width: 600px;
+	display: flex;
+	justify-content: flex-end;
+}
+
+ul {
+	list-style: none;
+}
+
+a,
+a:visited {
+	text-decoration: none;
+	color: #606060;
+}
+
+hr.popup-hr {
```

Comment:
Avoid writing too specific CSS selectors. Also, avoid styling by HTML tag or `#id` [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)